fb_instant_articles.module

```diff
@@ -18,6 +18,16 @@ function fb_instant_articles_menu() {
     'file path' => drupal_get_path('module', 'system'),
   );
 
+  $items['admin/config/services/fb-instant-articles/settings'] = array(
```

Comment:
@simonengelhardt  Oh, do you mean from the modules page? If so yep that's because I need to update the info file config path. Will do now, nice catch!